README.md

```diff
@@ -54,6 +54,7 @@ Build the binary.
 $ mkdir -p $GOPATH/src/github.com/kubernetes-incubator/external-dns
 $ cd $GOPATH/src/github.com/kubernetes-incubator/external-dns
 $ git clone https://github.com/kubernetes-incubator/external-dns.git .
+$ glide install --strip-vendor
```

Comment:
@mikkeloscar any thoughts? I believe it's not needed as it finds all materialized dependencies in `vendor/`